index.html

```diff
@@ -2,479 +2,515 @@
-                            Personal con formacion y con un enfoque de
-                            transparencia y honestidad que garantiza la satisfaccion
-                            del cliente.
-                        </p>
-                    </div>
-                </div>
-                <!--oooooooooooooooooooooooooooooooooo-->
-
-                <div id="cont-about-us">
-                    <div class="icon">
-                        <img src="images/icons-green.png">
-                    </div>
-                    <div id="description">
-                        <h2>VERDE</h2>
-                        <p>
-                            Doncella esta motivada por la ecologia y la vida saludable,
-                            ofreciendo siempre soluciones con un alto grado de conciencia
-                            ambiental.
-                        </p>
-                    </div>
-                </div>
+      <p>Todos nuestros servicios estan disponibles a nivel nacional en Republica Dominicana.</p>
 
-            </div>
-            <!--oooooooooooooooooooooooooooooooooo-->
-
-            <div id="container-mvv">
-                <h2 class="h2-green">MISION, VISION y VALORES</h2>
-                <div id="cont-flex-mvv">
-                    <div id="mission-view-values" class="center" style="margin: 0 auto">
-                        <div id="card-mission" class="card">
-                            <h2>Mision</h2>
-                            <p>Ofrecer a residencias y empresas, servicios y
-                                productos de limpieza con conciencia ambiental.
-                            </p>
-                        </div>
-                        <div id="card-view" class="card">
-                            <h2>Vision</h2>
-                            <p>Mantener el liderazgo en el mercado y evolucionar como
-                                empresa lider en servicios y productos de limpieza verde.
-                            </p>
-                        </div>
-                        <div id="card-values" class="card">
-                            <h2>Valores</h2>
-                            <ul>
-                                <li>Satisfaccion al Cliente</li>
-                                <li>Creatividad</li>
-                                <li>Honestidad</li>
-                                <li>Liderazgo</li>
-                                <li>Trato Humano</li>
-                            </ul>
-                        </div>
-                    </div>
-
-                    <!--oooooooooooooooooooooooooooooooooo-->
-
-                    <button type="button" id="btn-mvv">QUIERES TRABAJAR CON NOSOTROS?</button>
-                </div>
-            </div>
-        </div>
     </div>
 
+    <!--PPPPPPPPPPPPPPPPPPPPPP_DIV_PAGINA_PPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPP-->
+    <div id="div-clients" class="center">
+      <a id="clients"></a>
+      <h1 id="white">CLIENTES</h1>
```

Comment:
Ids are meant to identify unique elements but I see you have multiple elements with the same id, please change them to a class instead of id